queries/proto/highlights.scm

```diff
@@ -1,15 +1,18 @@
 [
-  "syntax"
-  "option"
-  "service"
-  "rpc"
-  "returns"
-  "message"
   "enum"
+  "extend"
+  "extensions"
+  "message"
   "oneof"
+  "option"
   "optional"
   "repeated"
   "reserved"
+  "returns"
```

Comment:
`@keyword.return`?

Reply:
this is not `return` as in a function return statement. it's a keyword for defining the return message type of a service RPC. for example:

```protobuf
service SearchService {
  rpc Search(SearchRequest) returns (SearchResponse);
}
```

Reply:
yeah I'm aware but imo it's similar enough to be warranted, and would allow a configurable yet clear distinction for users

Reply:
alrighty I can get on board with that. updated.